scripts/create-seed-fixture-heartwood

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#function cleanup {
```

Comment:
you can clean up all comments, since we have the original script in repo

Reply:
00cb65a6dea2a9db792f10d5db8d7a43366e2b71